bandit/plugins/pytorch_load_save.py

```diff
@@ -59,10 +64,14 @@ def pytorch_load_save(context):
     if all(
         [
             "torch" in qualname_list,
-            func in ["load", "save"],
-            not context.check_call_arg_value("map_location", "cpu"),
+            func == "load",
```

Comment:
Note, the title of the check still refers to save

Reply:
ahh good point, let me fix.